scss/_print.scss

```diff
@@ -18,18 +24,37 @@
     text-decoration: underline;
   }
 
+  a[href]::after {
+    content: " (" attr(href) ")";
+  }
+
   abbr[title]::after {
     content: " (" attr(title) ")";
   }
 
+  //
```

Comment:
All right, I wonder how we could mark those as such for future reference.

How about the border variable?
On Jan 25, 2016 10:26, "Chris Rebert" <notifications@github.com> wrote:

> In scss/_print.scss
> <https://github.com/twbs/bootstrap/pull/19018#discussion_r50664424>:
>
> >    abbr[title]::after {
> >      content: " (" attr(title) ")";
> >    }
> >
> > +  //
>
> Ditto
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twbs/bootstrap/pull/19018/files#r50664424>.
>